api/integrations/github/github.py

```diff
@@ -1,70 +1,27 @@
 import logging
 import typing
-from dataclasses import dataclass
 
-import requests
 from core.helpers import get_current_site_url
-from django.conf import settings
 from django.utils.formats import get_format
 
 from features.models import Feature, FeatureState, FeatureStateValue
-from integrations.github.client import generate_token
 from integrations.github.constants import (
     DELETED_FEATURE_TEXT,
+    DELETED_SEGMENT_OVERRIDE_TEXT,
     FEATURE_ENVIRONMENT_URL,
     FEATURE_TABLE_HEADER,
     FEATURE_TABLE_ROW,
-    GITHUB_API_URL,
     LINK_FEATURE_TITLE,
     LINK_SEGMENT_TITLE,
     UNLINKED_FEATURE_TEXT,
     UPDATED_FEATURE_TEXT,
 )
-from integrations.github.models import GithubConfiguration
 from webhooks.webhooks import WebhookEventType
 
-logger = logging.getLogger(__name__)
-
-
-@dataclass
-class GithubData:
-    installation_id: str
-    feature_id: int
-    feature_name: str
-    type: str
-    feature_states: typing.List[dict[str, typing.Any]] | None = None
-    url: str | None = None
-    project_id: int | None = None
-
-    @classmethod
-    def from_dict(cls, data_dict: dict) -> "GithubData":
-        return cls(**data_dict)
-
-
-def post_comment_to_github(
-    installation_id: str, owner: str, repo: str, issue: str, body: str
-) -> typing.Optional[typing.Dict[str, typing.Any]]:
-    try:
-        token = generate_token(
-            installation_id,
-            settings.GITHUB_APP_ID,
-        )
-
-        url = f"{GITHUB_API_URL}repos/{owner}/{repo}/issues/{issue}/comments"
-        headers = {
-            "Accept": "application/vnd.github.v3+json",
-            "Authorization": f"Bearer {token}",
-        }
+from .dataclasses import GithubData
+from .models import GithubConfiguration
```

Comment:
Let's avoid relative imports:

```suggestion
from integrations.github.dataclasses import GithubData
from integrations.github.models import GithubConfiguration
```

Reply:
Done